plugins/org.eclipse.elk.core.service/src/org/eclipse/elk/core/service/LayoutConfigurationManager.java

```diff
@@ -68,10 +67,10 @@ public LayoutAlgorithmData getAlgorithm(final ILayoutConfigurationStore config)
      * @param config a layout configuration store associated to a graph element
      * @return a list of supported options
      */
-    public List<LayoutOptionData> getSupportedOptions(final ILayoutConfigurationStore config) {
+    public Set<LayoutOptionData> getSupportedOptions(final ILayoutConfigurationStore config) {
         LayoutMetaDataService layoutDataService = LayoutMetaDataService.getInstance();
         
-        List<LayoutOptionData> optionData = new LinkedList<LayoutOptionData>();
+        Set<LayoutOptionData> optionData = new HashSet<LayoutOptionData>();
```

Comment:
I'd use a LinkedHashSet here. It may not be relevant for the Layout view because it sorts the entries, but better be safe.